City-of-Helsinki/helsinki-paatokset
# PP-225: Add discussion minutes to board transcripts

Add discussion minutes link to the bottom of the single minute page.

To test:
- Add new media of type minutes_of_the_discussion
- Make sure you have data from paatokset_meeting, paatokset_agenda_item and paatoket_meeting_document migrations
- Check a policymaker's document page, eg. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto/asiakirjat. Link the newly added media to on of the meetings that has documents.
- View the minute page for the meeting by selecting it from /asiakirjat list. The media file should be displayed at the bottom of the page. Clicking it should open the PDF file.
